docs/tutorials/join-dal-baker/run-baker.md

```diff
@@ -2,23 +2,54 @@
 title: "Step 4: Run an Octez baking daemon"
 authors: Tezos core developers, Tim McMackin
 last_update:
-  date: 2 December 2024
+  date: 19 December 2024
 ---
 
-Now that you have a DAL node, you can run a baking daemon that can attest to DAL data or restart an existing baking daemon to connect it to the DAL node.
+Now that you have a layer 1 node and a DAL node, you can run a baking daemon that can create blocks and attests to DAL data.
+If you already have a baking daemon, you can restart it to connect to the DAL node.
+
+1. Optional: Set up a remote signer to secure the keys that the baker uses as described in [Signer](https://tezos.gitlab.io/user/key-management.html#signer) in the Octez documentation.
 
 1. To run a baking daemon that connects to the DAL, start it as usual and pass the URL to your DAL node to it with the `--dal-node` argument:
 
    ```bash
    octez-baker-PsParisC run with local node "$HOME/.tezos-node" my_baker --liquidity-baking-toggle-vote pass --adaptive-issuance-vote on --dal-node http://127.0.0.1:10732
    ```
 
+   Note that the command for the baker depends on the protocol version.
+   This example uses the ParisC protocol, so the command starts with `octez-baker-PsParisC`.
```

Comment:
Is there a way to use a variable here for the protocol suffix that gets instantiated by Docusaurus at build time, to avoid changing the tutorial  at a protocol shift?

Reply:
There are a few ways to do this -- I'll add a note as part of our discussion about a list of to-dos for each protocol upgrade.